<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Perfil</class>
 <widget class="QWidget" name="Perfil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hLayoutFunction"/>
   </item>
   <item>
    <widget class="QLabel" name="labelPunto">
     <property name="text">
      <string>Pixel: _ Valor: _</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButtonPerfil">
       <property name="text">
        <string>Perfil</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtondPerfil">
       <property name="text">
        <string>d(Perfil)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonPerfilSuave">
       <property name="text">
        <string>Perfil Suave</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtondPerfilSuave">
       <property name="text">
        <string>d(PerfilSuave)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
